zmdocument.cls

```diff
@@ -247,10 +241,8 @@
    \setcounter{@authorsNumber}{\the@listSize}
    \setcounter{@columnNumber}{\the\numexpr(1 + \the@authorsNumber / \the@authorsByColumn)\relax}
    \thispagestyle{empty}
-   \pagecolor{bgtitle}
-   \color{texttitle}
    \begin{flushright}
-      \Large \bfseries \renewcommand*{\do}[1]{\href{\@authorlink/##1/}{\bsc{##1}}\\}
+      \Large \bfseries \renewcommand*{\do}[1]{\href{\@authorlink/##1/}{\color{internalLinkColor}\bsc{##1}}\\}
```

Comment:
I thought about `\color{texttitle}`. But maybe you should define a `authorLinkColor`.